### PR TITLE
Marius/ziplibfix306

### PR DESCRIPTION
**Pull request by nkundiushuti**
this re-implements ziplib extractall to correctly extract files archived on Windows systems with cp437 encoding
It has been tested with orchset and guitarset

### CODE REVIEW

**Comment by nkundiushuti** [comment]
@genisplaja it would be great to test this on IRMAS with test_full_dataset to see whether it solved your problem. just copy paste the download_utils.py but revert the changed before committing/do not commit this file.

**Review by genisplaja — APPROVED** [review]
It works perfectly for me! The test_full_dataset of IRMAS passed with no problem, so the encoding issue is solved! Sorry for the delay, it took some time for the whole dataset to download and validate. 
Thank you very much Marius for this fix!!

File: tests/test_download_utils.py [hunk 1]
```diff
@@ -207,3 +208,10 @@ def test_download_tar_file(mocker, mock_file, mock_untar):
     mock_untar.assert_called_once_with("foo", cleanup=True)
     if os.path.exists('a'):
         shutil.rmtree('a')
+
+def test_extractall_unicode(mocker,mock_file,mock_unzip):
```
Comment (rabitt) [inline tests/test_download_utils.py:212]: Could you add an additional test file that has content which breaks the previous version?

**Review by rabitt — CHANGES_REQUESTED** [review]

File: tests/test_download_utils.py (hunk 1 above)
Comment (nkundiushuti) [reply tests/test_download_utils.py:212]: I do not have access to a Windows computer at the moment but I will try to find out one to create a similar archive to the IRMAS one

Comment (nkundiushuti) [reply tests/test_download_utils.py]: done!

**Comment by nkundiushuti** [comment]
I am going to close this and open a new branch on mirdata. I think this branch got messed up because of some previous unstaged commits in the MTG fork of mirdata

**Pull request closed without merging**